Cleaning up rule c249d5 "Device motion based changes to the content can be disabled" (#1393)
act-rules/act-rules.github.io: _rules/device-motion-disabled-c249d5.md

For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target one of the following is true:

- **no changes:** The registered event does not cause [changes to the content][changes in content] of the [web page][] within a 1 minutes time span of the [event firing][event firing]; or

> - **disabled:** There is at least one set of [instruments][instrument], where each [instrument][] is in the same [web page][] of the registered event or can be found in a [clearly labeled location][] from that [web page][]. The set of [instruments][instrument] can be used to prevent [changes to the content][changes in content] of the [web page][] resulting from the event within a 1 minutes time span of the [event firing][].

Reviewer: Not completely related to this PR, but this item could benefit from https://github.com/act-rules/act-rules.github.io/blob/dca29f7ac2cbde4023d389448b483872a18e40ae/pages/glossary/set-of-clearly-labelled-instruments.md and https://github.com/act-rules/act-rules.github.io/blob/dca29f7ac2cbde4023d389448b483872a18e40ae/pages/glossary/block-event.md

Using "block event" might also help to solve #1394 without getting an awful text:
"There is on [set of clearly labelled instruments][] to [block the event][] for at least YYY minutes after activating the instruments"
or something in that line.

Author: I completely agree with the proposed changes.
I guess we can wait for those definitions to be merged and then update the "device motion" rules.

Reviewer: Sounds like a plan.

Reviewer: We can use the "new" definitions for these notions.
```suggestion
- **disabled:** There is at least one [set of clearly labelled instruments][] to [block the event][blocked event][] for at least 1 minute.
```
(plus adding the refs)
